<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Параметры РУС и РУД</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="checkBoxKey">
      <property name="text">
       <string>Клавиатура</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainStatusLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="parameters">
      <item>
       <layout class="QHBoxLayout" name="coordinates">
        <item>
         <widget class="QLabel" name="x_label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>X =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="x_value">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_1">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="y_label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Y=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="y_value">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="z_label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Z = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="z_value">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ped_label">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>PED = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ped_value">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Площадки = </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="torm_value">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="other_axes">
        <item>
         <widget class="QLabel" name="ant_elev">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>ANT ELEV=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ant_elev_v">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="man_rng">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>MAN RNG=</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="man_rng_v">
          <property name="text">
           <string>N/A</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttons">
        <item>
         <layout class="QVBoxLayout" name="rusButtons">
          <item>
           <widget class="QLabel" name="rus_buttons">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Кнопки РУС</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="rusButtonsTable">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAsNeeded</enum>
            </property>
            <property name="autoScroll">
             <bool>false</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>false</bool>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="dragDropMode">
             <enum>QAbstractItemView::NoDragDrop</enum>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>6</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>7</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>8</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>9</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Название</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Статус</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>Триггер</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>Отжата</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>WPN REL</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>Отжата</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>NWS</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>Отжата</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>Bottom</string>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string>Отжата</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>САУ откл.</string>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string>Отжата</string>
             </property>
            </item>
            <item row="5" column="0">
             <property name="text">
              <string>Trim</string>
             </property>
            </item>
            <item row="5" column="1">
             <property name="text">
              <string>CENTER / CENTER</string>
             </property>
            </item>
            <item row="6" column="0">
             <property name="text">
              <string>Left 4-pos</string>
             </property>
            </item>
            <item row="6" column="1">
             <property name="text">
              <string>CENTER / CENTER</string>
             </property>
            </item>
            <item row="7" column="0">
             <property name="text">
              <string>Right 4-pos</string>
             </property>
            </item>
            <item row="7" column="1">
             <property name="text">
              <string>CENTER / CENTER</string>
             </property>
            </item>
            <item row="8" column="0">
             <property name="text">
              <string>Thumb 4-pos</string>
             </property>
            </item>
            <item row="8" column="1">
             <property name="text">
              <string>CENTER / CENTER</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="rudButtons">
          <item>
           <widget class="QLabel" name="rud_buttons">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>Кнопки РУД</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="rudButtonsTable">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="tabKeyNavigation">
             <bool>false</bool>
            </property>
            <property name="showDropIndicator" stdset="0">
             <bool>false</bool>
            </property>
            <property name="dragDropOverwriteMode">
             <bool>false</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>100</number>
            </attribute>
            <row>
             <property name="text">
              <string>1</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>2</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>3</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>4</string>
             </property>
            </row>
            <row>
             <property name="text">
              <string>5</string>
             </property>
            </row>
            <column>
             <property name="text">
              <string>Название</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Статус</string>
             </property>
            </column>
            <item row="0" column="0">
             <property name="text">
              <string>Enable</string>
             </property>
            </item>
            <item row="0" column="1">
             <property name="text">
              <string>Отжата</string>
             </property>
            </item>
            <item row="1" column="0">
             <property name="text">
              <string>UNCAGE</string>
             </property>
            </item>
            <item row="1" column="1">
             <property name="text">
              <string>CENTER</string>
             </property>
            </item>
            <item row="2" column="0">
             <property name="text">
              <string>DOG FIGHT</string>
             </property>
            </item>
            <item row="2" column="1">
             <property name="text">
              <string>CENTER</string>
             </property>
            </item>
            <item row="3" column="0">
             <property name="text">
              <string>SPD BRK</string>
             </property>
            </item>
            <item row="3" column="1">
             <property name="text">
              <string>CENTER</string>
             </property>
            </item>
            <item row="4" column="0">
             <property name="text">
              <string>HF / IFF</string>
             </property>
            </item>
            <item row="4" column="1">
             <property name="text">
              <string>CENTER / CENTER</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1157</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
